server/index.ts

```diff
@@ -1,17 +1,17 @@
 import { Server } from "boardgame.io/server";
 import path from "path";
 import serve from "koa-static";
-import { default as ScytheBidderGame } from "./src/scythe-bidder/game";
+import { default as ScytheBidderGame } from "../src/scythe-bidder/game";
 
 const server = Server({ games: [ScytheBidderGame] });
-const PORT = process.env.PORT || 8000;
+const PORT = Number(process.env.PORT) || 8000;
```

Comment:
Is casting it to a Number OK? Also, could we use an env variable for `server` in `lobby.tsx`? I keep committing the dev version on accident...